### PR TITLE
DSD-860: Updating the Image's caption text to font size 12px

### PR DESCRIPTION
**Pull request by EdwinGuzman**
Fixes JIRA ticket [DSD-860](https://jira.nypl.org/browse/DSD-860)

## This PR does the following:

- Uses the `text.tag` design token to update the `Image`'s caption and credit.

## How has this been tested?

Storybook and updated snapshot tests.

## Accessibility concerns or updates

- N/A

### Checklist:

- [ ] I have updated the Storybook documentation accordingly.
- [ ] I have added relevant accessibility documentation for this pull request.
- [ ] All new and existing tests passed.

### Front End Review:

- [ ] View [the example in Storybook]()

### CODE REVIEW

File: src/theme/components/image.ts [hunk 1]
```diff
@@ -85,6 +85,7 @@ const CustomImage = {
     },
     figcaption: {
       fontStyle: "italic",
+      fontSize: "text.tag",
```
Comment (oliviawongnyc) [inline src/theme/components/image.ts:88]: Why are we switching which element the fontSize is added to?

Comment (EdwinGuzman) [reply src/theme/components/image.ts:88]: The element where `figcaption` is applied is the parent of the element where `captionWrappers` is applied. So it covers both "captionWrappers".

Comment (oliviawongnyc) [reply src/theme/components/image.ts:88]: Ah, okay. Thank you!

**Review by oliviawongnyc — APPROVED** [review]

**Review by bigfishdesign13 — APPROVED** [review]
Looks good.  Simple.

**Pull request merged**